sys-process/rtirq/rtirq-20180209.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit systemd
+
+DESCRIPTION="Modify realtime scheduling policy and priority of IRQ handlers"
+HOMEPAGE="http://www.rncbc.org/archive/#rtirq"
+
+SRC_URI="http://www.rncbc.org/archive/${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_prepare() {
+	# Correct config file path.
+	sed -i -e "s:^\(RTIRQ_CONFIG\=\)\(.*\):\1/etc/conf.d/rtirq:" ${PN}.sh || die
+	sed -i -e "s:/etc/sysconfig/rtirq:/etc/conf.d/rtirq:" ${PN}.conf || die
+
+	default
+}
+
+src_install(){
+	dosbin ${PN}.sh
+	doinitd "${FILESDIR}"/${PN}
+	newconfd ${PN}.conf ${PN}
+	systemd_dounit ${PN}.service
+}
+
+pkg_postinst(){
+	if [[ "$(rc-config list default | grep rtirq)" = "" ]] ; then
```

Comment:
1. I think you should check for `[[ -z ${ROOT} ]]` for this. You don't want to invoke rc-config when installing not to the main system.
2. This doesn't account for the possibility of OpenRC not being used or even installed (i.e. it will throw some ugly error probably). I'm sorry but I don't know how to check for OpenRC being booted.
3. I don't think `rc-config list` output is really suitable for machine processing.
4. I don't think it's really reasonable to have every package inform the user that he needs to enable init.d script.

Reply:
I opted for removing the extra code. With your comments in mind, It is not worth the effort of trying to help the user not forget to call `rc-update add rtirq default`.